Source/CarthageKit/Xcode.swift

```diff
@@ -476,7 +476,8 @@ public struct BuildSettings {
 	/// Upon .success, sends one BuildSettings value for each target included in
 	/// the referenced scheme.
 	public static func loadWithArguments(arguments: BuildArguments) -> SignalProducer<BuildSettings, CarthageError> {
-		let task = xcodebuildTask("-showBuildSettings", arguments)
+		// Including build action "clean" here, because otherwise Xcode 8 (at least in betas) will hang
+		let task = xcodebuildTask(["clean", "-showBuildSettings"], arguments)
```

Comment:
Can you include more detail in the comment? It'd be nice if it mentioned (1) that the hang occurs when the project contains a core data model and (2) the devforums post.